lib/ethpy/ethpy/test_fixtures/local_chain.py

```diff
@@ -59,16 +59,13 @@ class LocalHyperdriveChain(NamedTuple):
     hyperdrive_factory_contract: Contract
     base_token_contract: Contract
 
-
-@pytest.fixture(scope="function")
-def local_hyperdrive_chain(local_chain: str) -> LocalHyperdriveChain:
+def create_hyperdrive_chain(rpc_uri: str, contract_addresses = None, initial_liquidity = None) -> LocalHyperdriveChain:
     """Initializes hyperdrive on a local anvil chain for testing.
-    Returns the hyperdrive contract address.
 
     Arguments
     ---------
-    local_chain: str
-        The `local_chain` test fixture that binds to the local anvil chain rpc url
+    rpc_uri: str
+        The URI of the local RPC node
```

Comment:
Docstrings of arguments please. Not sure why contract_addreses and initial_liquidity is being passed in

Reply:
my whole rationale for using this is to allow me to change the initial liquidity amount. I use it like this, to deploy the minimum liquidity possible: `create_hyperdrive_chain("http://127.0.0.1:8546", initial_liquidity=int(20*1e18))`

when the contract size wasn't allowed me to deploy the hyperdrive factory, I tried an alternate path: use the deployed factory to deploy a new hyperdrive pool. that's what contract_addresses is for. it passes in the existing deployment addresses, to look up the factory, and use it to deploy a new hyperdrive. it seems like saving that step would be nice to have. although the factory contains a lot of parameters you can't overwrite, like fees. I never got this flow to work, since apparently you also have to deploy a new vault for a new pool, which is currently lumped into `deploy_hyperdrive_factory`. should we remove this path for now, or leave it as a TODO?

Reply:
removed unfinished flow, tracking it here: https://github.com/delvtech/elf-simulations/issues/907

Reply:
added docstrings inb822697107835a8f75c2f2e04a39d99dc0bb701f